apps/sasquatch/src/main/res/xml/settings.xml

```diff
@@ -35,6 +35,12 @@
         <CheckBoxPreference
             android:key="@string/appcenter_auto_page_tracking_key"
             android:title="@string/appcenter_auto_page_tracking_title" />
+        <CheckBoxPreference
+            android:key="@string/appcenter_analytics_session_tracker_state_key"
+            android:title="@string/appcenter_analytics_session_tracker_title" />
+        <Preference
+            android:key="@string/appcenter_analytics_start_session_key"
```

Comment:
shouldn't start session button go to the main screen?

Reply:
nope, it should be here